src/viur/core/modules/file.py

```diff
@@ -422,6 +444,21 @@ class File(Tree):
 
     # Helper functions currently resist here
 
+    @staticmethod
+    def get_bucket(dlkey: str) -> google.cloud.storage.bucket.Bucket:
+        """
+        Retrieves a Google Cloud Storage bucket for the given dlkey.
+        """
+        if dlkey and dlkey.endswith(PUBLIC_DLKEY_POSTFIX):
+            if public_bucket := GOOGLE_STORAGE_CLIENT.lookup_bucket(f"""public-dot-{_PROJECT_ID}"""):
+                return public_bucket
+
+            raise ValueError(
+                f"""The bucket 'public-dot-{_PROJECT_ID}' does not exist! Please create it with ACL access."""
+            )
+
+        return GOOGLE_STORAGE_CLIENT.lookup_bucket(f"""{_PROJECT_ID}.appspot.com""")
```

Comment:
```suggestion
def get_bucket(dlkey: str) -> google.cloud.storage.bucket.Bucket:
	"""
	Retrieves a Google Cloud Storage bucket for the given dlkey.
	"""
	if dlkey and dlkey.endswith(PUBLIC_DLKEY_POSTFIX):
		if public_bucket := conf.file_buckets.get(f"""public-dot-{_PROJECT_ID}"""):
			return public_bucket
		if public_bucket := GOOGLE_STORAGE_CLIENT.lookup_bucket(f"""public-dot-{_PROJECT_ID}"""):
			conf.file_buckets[f"""public-dot-{_PROJECT_ID}"""] = public_bucket
			return public_bucket

		raise ValueError(
			f"""The bucket 'public-dot-{_PROJECT_ID}' does not exist! Please create it with ACL access."""
		)
	
	if private_bucket := conf.file_buckets.get(f"""{_PROJECT_ID}.appspot.com"""):
		return private_bucket
	
	private_bucket = conf.file_buckets.get(f"""{_PROJECT_ID}.appspot.com"""):
	conf.file_buckets[f"""{_PROJECT_ID}.appspot.com"""] = private_bucket
	return private_bucket
```
We can solve it like this than we can save us lookups we only need `file_buckets` in `conf`

Reply:
Hello @ArneGudermann, we can either put it into conf or make it global in file.py... latter one is closer to the implementation before the change. I wouldn't put it into the config.

Reply:
@ArneGudermann are you happy with e19a204c09ad2d5f399bfaba6f8e920b11ce15f1?